hack/make.ps1

```diff
@@ -126,6 +126,25 @@ Function Check-InContainer() {
         Write-Host ""
         Write-Warning "Not running in a container. The result might be an incorrect build."
         Write-Host ""
+        return $False
+    }
+    return $True
+}
+
+# Utility function to warn if the version of go is correct. Used for local builds
+# outside of a container where it may be out of date with master.
+Function Verify-GoVersion() {
+    Try {
+        $goVersionDockerfile=(Get-Content ".\Dockerfile" | Select-String "ENV GO_VERSION").ToString().Split(" ")[2]
```

Comment:
nit: it would be better to use regexp for an accidental case where Dockerfile contains extra spaces (e.g. `ENV[SPC][SPC]GO_VERSION`)